code/aepcomposeui/src/main/java/com/adobe/marketing/mobile/aepcomposeui/aepui/style/AEPTextExt.kt

```diff
@@ -0,0 +1,24 @@
+package com.adobe.marketing.mobile.aepcomposeui.aepui.style
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextAlign
+
+// only needed when we support server side styling
+fun AEPText.getComposeTextStyle(): TextStyle {
```

Comment:
Can we remove this class? There's no support for server side styling in AJO. We can add it when we need to